go.mod

```diff
@@ -37,7 +37,7 @@ require (
 	github.com/pborman/uuid v1.2.1
 	github.com/pkg/errors v0.9.1
 	github.com/portworx/pds-api-go-client v0.0.0-20231102112445-993d38984eae
-	github.com/portworx/px-backup-api v1.2.2-0.20240513165102-9eda9e2fc169
+	github.com/portworx/px-backup-api v1.2.2-0.20240717111040-e2486ca8612f
```

Comment:
Please use 2.8.0 for vendoring. Master is not upto date.

Reply:
Updated to latest commit from 2.8.0 `401f4e76b9795a07c1ec8c2f0b91ee0b4845300a` ,